### PR TITLE
Updates the Gemspec for Wisper 2.0

### PR DESCRIPTION
**Pull request by rringler**
### Description
* Updates the Gemspec for Wisper 2.0.
* Fixes a Typo in the spec directory
* Updates (loosens) the version requirement for the Rubocop gem
* Simplifies `Wisper::Resque` (in-lining the `ResqueBroadcaster` class) for better parity with the `wisper-sidekiq` gem.
* Bumps the `Wisper::Resque::VERSION` constant to '0.2.0'

### CODE REVIEW

File: spec/lib/whisper/resque_spec.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+RSpec.describe Wisper do
+  describe Wisper::ResqueBroadcaster do
```
Comment (bzurkowski) [inline spec/lib/whisper/resque_spec.rb]: Please, wrap it into single line: 

```ruby
RSpec.describe Wisper::ResqueBroadcaster do
  # ...
end
```

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -9,17 +9,18 @@
...
-  config.filter_run_when_matching :focus
+  config.filter_run :focus
+  config.run_all_when_everything_filtered = true
 
   config.disable_monkey_patching!
 
   config.warnings = true
 
-  config.default_formatter = 'doc' if config.files_to_run.one?
+  if config.files_to_run.one?
```
Comment (bzurkowski) [inline spec/spec_helper.rb]: [Rubocop](https://github.com/rubocop-hq/ruby-style-guide#if-as-a-modifier) says: ```Favor modifier if usage when having a single-line body.```

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -9,17 +9,18 @@
...
+    config.default_formatter = 'doc'
+  end
 
-  config.profile_examples = 10
+  config.profile_examples = 0
```
Comment (bzurkowski) [inline spec/spec_helper.rb]: I'd rather leave the default value.

**Review by bzurkowski — CHANGES_REQUESTED** [review]

**Comment by bzurkowski** [comment]
Also, please split this two concerns (code simplification and dependency update) into separate  commits or at least modify commit message. Thank you!

File: spec/spec_helper.rb (hunk 2 above)
Comment (rringler) [reply spec/spec_helper.rb]: I was keeping parity with the `wisper-sidekiq` gem, but I don't think that project is enforcing style with rubocop. I will update this.

Comment (bzurkowski) [reply spec/spec_helper.rb]: I'll setup Travis with Rubocop + RSpec afterwards.

**Comment by rringler** [comment]
Thank you!

**Pull request merged**